### PR TITLE
Merge devel into master

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

File: packages/lesswrong/components/editor/EditorFormComponent.tsx [hunk 1]
```diff
@@ -593,8 +592,7 @@ class EditorFormComponent extends Component<EditorFormComponentProps,EditorFormC
   renderVersionSelect = () => {
     const { classes, document, currentUser } = this.props
 
-    if (!userHasCkEditor(currentUser)) return null
-
+    if (!currentUser?.isAdmin) return null
```
Comment (Raemon) [inline packages/lesswrong/components/editor/EditorFormComponent.tsx:595]: It looks like we'd been giving version select to everyone who had enabled ckEditorBeta before, and now we are removing that? 

I think we should probably just give everyone version select, although not necessarily in this particular deploy

Comment (Discordius) [reply packages/lesswrong/components/editor/EditorFormComponent.tsx:595]: We definitely shouldn't give anyone the current version select component. It's horribly broken, sometimes deletes your content, and doesn't even work reliably for admins. We should at some point actually push a functioning version-select component, but that's a big push that will probably take multiple days.

Comment (darkruby501) [reply packages/lesswrong/components/editor/EditorFormComponent.tsx:595]: Let me know when this merge actually happens. I need to do stuff for Tagging Dashboard switchover.

File: packages/lesswrong/components/users/UsersNameDisplay.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const styles = (theme: ThemeType): JssStyles => ({
 })
 
 interface ExternalProps {
-  user: UsersMinimumInfo,
+  user: UsersMinimumInfo|null|undefined,
   nofollow?: boolean,
```
Comment (Raemon) [inline packages/lesswrong/components/users/UsersNameDisplay.tsx:42]: so sad. But, seems fine.

File: packages/lesswrong/lib/crud/withSingle.ts [hunk 3]
```diff
@@ -119,7 +119,7 @@ export function useSingle<FragmentTypeName extends keyof FragmentTypes>({
   extraVariablesValues?: any,
   skip?: boolean,
 }): {
-  document: FragmentTypes[FragmentTypeName]
+  document: FragmentTypes[FragmentTypeName],
```
Comment (Raemon) [inline packages/lesswrong/lib/crud/withSingle.ts:122]: man how was this even working before?

Comment (jimrandomh) [reply packages/lesswrong/lib/crud/withSingle.ts:122]: Commas are required when entries in an object are on the same line, but optional (in the same way semicolons are optional) when split between lines.)

**Review by Raemon — APPROVED** [review]

**Pull request merged**